String_Lab.js

```diff
@@ -0,0 +1,184 @@
+
+// 1. console.log the first character of a string.
+let string = "Lesson";
+console.log("Printing the first character of the word ", string);
+console.log(string[0]);
+console.log("---------------------------------- \n");
+
+
+// 2. console.log the length of a string.
+console.log(`The lenght of ${string} is ${string.length}`)
+console.log("---------------------------------- \n");
+
+
+// 3. console.log the last character of any string.
+console.log(`The last character of the word ${string} is ${string[string.length - 1]}`);
+console.log("---------------------------------- \n");
+
+
+// 4. Create a code block that takes a single string variable, and returns
+// a copy of the string with the last letter capitalized. For example:
+
+// 'jimmy'
+// => 'jimmY'
+console.log(`Capitalizing the last character of the word ${string} :`);
+console.log("Methode1 : ")
+console.log(string.slice(0, string.length - 1) + string[string.length - 1].toUpperCase());
+console.log("Methode2 : ")
+console.log(`${string}\b${string[string.length - 1].toUpperCase()}`);
```

Comment:
The use of `\b` (backspace) for string manipulation yields the result we are looking for and I'm glad you found such special character but is not used conventionally. Think of the result of this expression `"Lesson\b\b".length`